aperturedb/Connector.py

```diff
@@ -396,7 +396,8 @@ def _connect(self):
                         f"The host name must match the certificate: {self.host} " + os.linesep + \
```

Comment:
* Should use `logger.exceeption` here to automatically report exception and traceback
* These three calls to logger should be one
* This call to assert might be better as `raise ssl.SSLError('...') from e`